frontend/querybuilder/querybuilder.ts

```diff
@@ -40,6 +40,10 @@ export function runApolloCodegen(): number {
   if (child.error) {
     throw child.error;
   }
+  if (child.status === null) {
+    // The child terminated due to a signal, just represent it as an error.
+    return 1;
+  }
```

Comment:
Upgrading `@types/node` apparently revealed the fact that `status` can be `null` if the process was stopped by a signal, so we deal w/ that here.